H/HDF5/build_tarballs.jl

```diff
@@ -6,226 +6,269 @@ const YGGDRASIL_DIR = "../.."
 include(joinpath(YGGDRASIL_DIR, "platforms", "mpi.jl"))
 
 name = "HDF5"
-version = v"1.14.3"
+version = v"1.14.5"
 
 # Collection of sources required to complete build
 sources = [
-    ArchiveSource("https://support.hdfgroup.org/ftp/HDF5/releases/hdf5-$(version.major).$(version.minor)/hdf5-$(version)/src/hdf5-$(version).tar.bz2",
-                  "9425f224ed75d1280bb46d6f26923dd938f9040e7eaebf57e66ec7357c08f917"),
-    DirectorySource("./bundled"),
+                   
+    ArchiveSource("https://support.hdfgroup.org/releases/hdf5/v$(version.major)_$(version.minor)/v$(version.major)_$(version.minor)_$(version.patch)/downloads/hdf5-$(version).tar.gz",
+                  "ec2e13c52e60f9a01491bb3158cb3778c985697131fc6a342262d32a26e58e44"),
+    DirectorySource("bundled"),
 ]
 
 # Bash recipe for building across all platforms
 script = raw"""
 cd ${WORKSPACE}/srcdir/hdf5-*
 
-if [[ ${target} == *-mingw* ]]; then
-    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/h5ls.c.patch
-    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/mkdir.patch
-    atomic_patch -p1 ${WORKSPACE}/srcdir/patches/strncpy.patch
-    cp ${WORKSPACE}/srcdir/headers/pthread_time.h "/opt/${target}/${target}/sys-root/include/pthread_time.h"
+atomic_patch -p1 ../patches/cmake-fortran.patch
+atomic_patch -p1 ../patches/mpi.patch
+
+cmake_options=(
+    -DCMAKE_BUILD_TYPE=Release
+    -DCMAKE_EXE_LINKER_FLAGS=-lsz      # help cmake link against the sz library
+    -DCMAKE_INSTALL_PREFIX=${prefix}
+    -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN}
+    -DMPI_HOME=${prefix}
+    -DALLOW_UNSUPPORTED=ON
+    -DBUILD_TESTING=OFF
+    -DHDF5_BUILD_CPP_LIB=ON
+    -DHDF5_BUILD_DOC=OFF
+    -DHDF5_BUILD_EXAMPLES=OFF
+    -DHDF5_BUILD_FORTRAN=ON
+    -DHDF5_BUILD_HL_LIB=ON
+    -DHDF5_BUILD_JAVA=OFF              # would require Java
+    -DHDF5_BUILD_TOOLS=ON
+    -DHDF5_ENABLE_DIRECT_VFD=ON
+    -DHDF5_ENABLE_HDFS=OFF             # would require Java
+    -DHDF5_ENABLE_MAP_API=ON
+    -DHDF5_ENABLE_MIRROR_VFD=ON
+    -DHDF5_ENABLE_PLUGIN_SUPPORT=OFF   # would require PLUGIN
+    -DHDF5_ENABLE_ROS3_VFD=ON
+    -DHDF5_ENABLE_SZIP_SUPPORT=ON
+    -DHDF5_ENABLE_THREADSAFE=ON
+    -DHDF5_ENABLE_Z_LIB_SUPPORT=ON
+    -DONLY_SHARED_LIBS=ON
+)
+# Help cmake find the sz library
+cp ${prefix}/cmake/libaec-config.cmake ${prefix}/cmake/szip-config.cmake
```

Comment:
Where's this file coming from?  From a dependency?  If so, you must delete it at the end otherwise it ends in _this_ package, which is wrong.

Reply:
Done.